src/OVAL/probes/SEAP/sch_pipe.c

```diff
@@ -275,12 +279,15 @@ ssize_t sch_pipe_recv (SEAP_desc_t *desc, void *buf, size_t len, uint32_t flags)
         sch_pipedata_t *data;
 	ssize_t         ret;
 
-        assume_d (desc != NULL, -1, errno = EFAULT;);
```

Comment:
This is a duplicate code with `sch_pipe_send`. Deduplicating may be out of scope, but at least put a TODO comment here.

Reply:
OK